BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/VerifiedAccount.java

```diff
@@ -0,0 +1,45 @@
+package nl.tudelft.ewi.ds.bankchain;
+
+/**
+ * @author Isha Dijcks
+ */
+public class VerifiedAccount {
+
+    private String publicKey;
```

Comment:
Please use the Verification class in the cryptography package. You could add a Date object to that to use here.